delta/plugins/storage/src/test/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/remotestorage/RemoteStorageClientFixtures.scala

```diff
@@ -1,14 +1,22 @@
-package ch.epfl.bluebrain.nexus.testkit.remotestorage
+package ch.epfl.bluebrain.nexus.delta.plugins.storage.remotestorage
 
-import ch.epfl.bluebrain.nexus.testkit.remotestorage.RemoteStorageDocker.{BucketName, RemoteStorageHostConfig}
-import org.scalatest.{BeforeAndAfterAll, Suite}
+import akka.actor.ActorSystem
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.remotestorage.RemoteStorageClientFixtures.{BucketName, RemoteStorageHostConfig}
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote.client.RemoteDiskStorageClient
+import ch.epfl.bluebrain.nexus.delta.plugins.storage.storages.operations.remote.client.RemoteDiskStorageClient.RemoteDiskStorageClientImpl
+import ch.epfl.bluebrain.nexus.delta.sdk.ConfigFixtures
+import ch.epfl.bluebrain.nexus.delta.sdk.auth.{AuthTokenProvider, Credentials}
+import ch.epfl.bluebrain.nexus.delta.sdk.http.{HttpClient, HttpClientConfig}
+import ch.epfl.bluebrain.nexus.delta.sdk.model.BaseUri
+import ch.epfl.bluebrain.nexus.testkit.scalatest.BaseSpec
+import org.scalatest.BeforeAndAfterAll
 
 import java.nio.file.attribute.PosixFilePermissions
 import java.nio.file.{Files, Path}
 import scala.concurrent.duration.DurationInt
 import scala.jdk.DurationConverters.ScalaDurationOps
 
-trait RemoteStorageDocker extends BeforeAndAfterAll { this: Suite =>
+trait RemoteStorageClientFixtures extends BeforeAndAfterAll with ConfigFixtures { this: BaseSpec =>
```

Comment:
Same approach as for Postgres and Elasticsearch for example, we don't care about the docker container itself but about a client we can build from it